src/ybus_calculations.jl

```diff
@@ -2,55 +2,85 @@
 Nodal admittance matrix (Ybus) is an N x N matrix describing a power system with N buses. It represents the nodal admittance of the buses in a power system.
 
 The Ybus Struct is indexed using the Bus Numbers, no need for them to be sequential
+
+The fields yft and ytf are the branch admittance matrices for the from-to and to-from branch admittances respectively. They are indexed with the branch numbers and the bus numbers.
+The branch numbers are sequential. The bus numbers are represented by fb, tb arrays of sequential bus indices (internal bus indices). 
```

Comment:
there are no branch numbers in PSY. We index with the branch names like in the Ybus